File: udemy/pascal/projetos/Vet_maior_menor.pas
program Vet_maior_menor;
var 
    notas : array[1..10] of real;
    maior, menor : real;
    i : Integer;
begin
    for i := 1 to 10 do
    begin
    write('infome o valor da posição ',i,' do vetor: ');
    read(notas[i]);
    if (i = 1) then
        begin
            maior := notas[i];
            menor := notas[i];
        end;
    if (notas[i]> maior) then
    maior := notas[i];
    if (notas[i]< menor) then
    menor := notas[i];
    end;
    write('maior: ', maior:6:2, ' menor: ', menor:6:2);
end.

File: udemy/java/vet_maior_menor/src/vet_maior_menor/Vet_maior_menor.java
package vet_maior_menor;
import javax.swing.JOptionPane;
public class Vet_maior_menor {
	public static void main(String[] args) {
		float vet[] = new float [5], maior = 0, menor = 0;
		int i = 0;
		for (; i<=4;i++) {
			vet[i] = Float.parseFloat(JOptionPane.showInputDialog("informe o valor da posição "+(i+1)+" do vetor"));
			if (i == 0) {
				maior = menor = vet[i];
			}
			if (vet[i] > maior) {
				maior = vet[i];
			}
			if (vet[i] < menor) {
				menor = vet[i];
			}
		}
		JOptionPane.showMessageDialog(null, "O maior valor do vetor é: "+maior+"\nO menor valor do vetor é: "+menor);
		
		
	}

}
